salt/orch/update.sls

```diff
@@ -119,6 +119,7 @@ pre-orchestration-migration:
   salt.state:
     - tgt: '{{ is_updateable_node_tgt }}'
     - tgt_type: compound
+    - expect_minions: false
```

Comment:
Looking at the final expression for the `target`:
``` 
is_updateable_node_tgt = '( ' + 'G@tx_update_reboot_needed:true' + ' and ' + \
  'not L@' + nodes_down|join(',') + \
  ' and G@roles:kube-master' + ' ) or ( ' + \
  'G@tx_update_reboot_needed:true' + ' and ' + \
  'not L@' + nodes_down|join(',') + \
  ' and G@roles:kube-minion' + ' )'
```
So, in theory, the `expect_minions` should not be necessary as the `target` is already excluding the `nodes_down`. So maybe the problem is in the `target` because, when computed, this expression is empty.